src/FluentValidation/Internal/MemberNameValidatorSelector.cs

```diff
@@ -79,6 +79,7 @@ public class MemberNameValidatorSelector : IValidatorSelector {
 			throw new ArgumentException($"Expression '{expression}' does not specify a valid property or field.");
 		}
 
-		return chain.ToString();
+		var propertyName = ValidatorOptions.Global.PropertyNameResolver(typeof(T), expression.GetMember(), expression);
+		return propertyName;

```

Comment:
I think this method can be simplified now to remove the creation of the chain entirely. 

```csharp
	private static string MemberFromExpression<T>(Expression<Func<T, object>> expression) {
		var propertyName = ValidatorOptions.Global.PropertyNameResolver(typeof(T), expression.GetMember(), expression);

		if (propertyName == null) {
			throw new ArgumentException($"Expression '{expression}' does not specify a valid property or field.");
		}

		return propertyName;
	}
```

Reply:
done